Assets/HoloToolkit/Build/Editor/BuildDeployTools.cs

```diff
@@ -52,13 +53,19 @@ public static bool BuildSLN(string buildDirectory, bool showConfDlg = true)
                         buildSuccess = true;
                     }
                 }
-            });
+            };
+
+            BuildSLNUtilities.RaiseOverrideBuildDefaults(ref buildInfo);
+
+            BuildSLNUtilities.PerformBuild(buildInfo);
 
             return buildSuccess;
         }
 
         public static string CalcMSBuildPath(string msBuildVersion)
         {
+            // TODO: robertes: If this is going to support msbuild 15 and beyond, it needs to be updated to use the COM or powershell interfaces to find msbuild.
+            //                 See: https://github.com/Microsoft/vssetup.powershell/wiki
```

Comment:
If I had to guess, it's to align it with the actual message text on the top line. We should probably remove/resolve most of these todos before we merge though.